TODO/women-and-mobile-games-learnings-for-developers.md

```diff
@@ -2,74 +2,74 @@
 > * 原文作者：[Tobias Knoke](https://medium.com/@tobias.knoke?source=post_header_lockup)
 > * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
 > * 本文永久链接：[https://github.com/xitu/gold-miner/blob/master/TODO/women-and-mobile-games-learnings-for-developers.md](https://github.com/xitu/gold-miner/blob/master/TODO/women-and-mobile-games-learnings-for-developers.md)
-> * 译者：
+> * 译者：[corresponding](https://github.com/corresponding)
 > * 校对者：
 
-# Women and mobile games: learnings for developers
+# 开发者须知：女性用户和手机游戏
 
-## The market opportunity in making mobile gaming more diverse, more inclusive, and more engaging
+## 得益于当前市场中的机会，手机游戏类型变得更加多样，内容变得更加有包容性，显得生机勃勃
 
 ![](https://cdn-images-1.medium.com/max/800/0*U3P6oAG_I-73IuY6.)
 
-There are now over two billion active Android devices which means more people are playing mobile games than ever before. The growth of the mobile games audience had led to an expansion of diversity in characteristics, needs, and motivations of those playing games. In our previous post [Who plays mobile games](https://medium.com/googleplaydev/who-plays-mobile-games-8b33f76bb6d8) we discussed the opportunity to think of players in terms of the needs met by playing games, rather than in terms of stereotypes and demographics. At the same time, there is a lot of conversation in the gaming community about gender and inclusivity, but relatively little research or discussion about the experiences of **women who play mobile games**.
+目前世界上有两亿多的安卓设备，与以前相比有更多的人会玩手机游戏。这些新增的用户会要求游戏角色的多样性，对游戏也有不同的需求。我们之前文章[Who plays mobile games](https://medium.com/googleplaydev/who-plays-mobile-games-8b33f76bb6d8)，我们抛开刻板的观念去讨论，玩家通过玩游戏来满足自己需求的可能性。 现在在游戏界有很多关于性别和包容的讨论，而关于女性手机游戏玩家的研究和讨论却很少。 
```

Comment:
The growth of the mobile games audience had led to an expansion of diversity in characteristics, needs, and motivations of those playing games. In our previous post [Who plays mobile games](https://medium.com/googleplaydev/who-plays-mobile-games-8b33f76bb6d8) we discussed the opportunity to think of players in terms of the needs met by playing games, rather than in terms of stereotypes and demographics.
这两句的翻译都有微小的错误。
其中“The growth of the mobile games audience had led to an”的意思是“用户的增长造成了……”而不是“新增的用户要求……”
“in terms of the needs met by playing games, rather than in terms of stereotypes and demographics.”的意思是“从用户需求的角度，而不是从用户总体特征和分类的角度”。
stereotypes and demographics我也不确定怎么翻译比较好。